### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by ManarOmari**
### In this project, we take the Awesome books application and convert it into a fully functional website. 

-  Add three new sections on top of the two sections we have already completed: Books list (done), Add book form (done), a new Contact info section, a Navigation bar, and a Footer.

### CODE REVIEW

File: new.js [hunk 1]
```diff
@@ -55,3 +55,38 @@ document.getElementById('add-book-form').addEventListener('submit', (e) => {
...
+  sectionBook.style.display = 'none';
+  sectionContact.style.display = 'none';
+  addNewBook.style.display = 'flex';
+});
+
+contact.addEventListener('click', () => {
+  sectionBook.style.display = 'none';
+  addNewBook.style.display = 'none';
+  sectionContact.style.display = 'block';
+});
```
Comment (AmaduKamara) [inline new.js:92]: - [ ] I cannot see the different sections appear upon clicking on any of the links in the navigation bar. For this milestone, you're required to add the navigation bar and when any of the links is clicked, the content for that section only appears on the screen. For instance, when you click the Add New link, the form to add books should appear and when you click the List link, the list of books stored in Local storage or books already added are shown on the screen. Kindly fix that 👍
- [ ] At the moment I cannot give feedback on your project if everything works fine as it should when nothing is being rendered to the screen.

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @ManarOmari & @Okechukwu-muokwugwo,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly
✅ No linter error
✅ Good descriptive PR 👍 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly work on the links to show the different contents as per a link clicked on the navbar 
just as explained in the comment below 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: new.js (hunk 1 above)
Comment (Jaymelfah) [reply new.js:92]: When we run the project on all our browsers the links work fine. We have just added id links to the links to see if any change will happen

**Review by dasileker — APPROVED** [review]
### **Hi @Okechukwu-muokwugwo** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**